.config/tsaoptions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "instanceUrl"        :  "https://msazure.visualstudio.com",
+  "areaPath"           :  "One\\TelReachSDKTeam",
+  "projectName"        :  "One",
+  "iterationPath"      :  "One",
+  "notificationAliases":  [
+    "Trask.Stalnaker@microsoft.com",
```

Comment:
@trask can i use appinsightsjavahelp@microsoft.com or create a separate dl to exclude Julia? She probably doesn't need to know about compliance alerts.

Reply:
yes, appinsightsjavahelp@microsoft.com sounds good 👍